Reports/router.js

```diff
@@ -1,4 +1,38 @@
 var express = require('express');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017');
```

Comment:
This makes me wonder, do we want to store the URL:PORT in some global variable elsewhere? @the-pat  thoughts?